libiocage/lib/events.py

```diff
@@ -92,7 +94,7 @@ def get_state_string(self,
         return pending
 
     @property
-    def type(self):
+    def type(self) -> str:
```

Comment:
why._.

Reply:
It's printed later. And it will be the reference for translations.